projboard/views.py

```diff
@@ -57,3 +58,27 @@ def home_page(request):
 
 def about_page(request):
     return render(request, 'about/about.html', {})
+
+
+def show_article(request, article_pk):
+
+    # TODO when Authintiaction is implemented => del this line and check wich user is loged in
+    user = User.get_user_by_nickname("User2")
+    article = Article.objects.get(pk=article_pk)
+
+    if request.method == 'POST':
+        try:
+            Like.objects.get(user_id=user.id, article_id=article_pk)
+            Like.delete_like(user_id=user.id, article_id=article_pk)
+
+        except ObjectDoesNotExist:
+            Like.create_like(user_id=user, article_id=article)
```

Comment:
Can you please explain? does the method check here check the like button click?
If not, wouldn't every entry to the article add\ delete the like?

Reply:
when client hit like if there is already a like i delete if not i add it .

Reply:
yes check like button click